recipes/starshot_card_block_type/config/core.entity_view_display.block_content.card.default.yml

```diff
@@ -0,0 +1,46 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - block_content.type.card
+    - field.field.block_content.card.body
+    - field.field.block_content.card.field_card_link
+    - field.field.block_content.card.field_card_media
+  module:
+    - link
+    - text
+id: block_content.card.default
+targetEntityType: block_content
+bundle: card
+mode: default
+content:
+  body:
+    type: text_default
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    weight: 0
+    region: content
+  field_card_link:
+    type: link
+    label: above
+    settings:
+      trim_length: 80
+      url_only: false
+      url_plain: false
+      rel: ''
+      target: ''
+    third_party_settings: {  }
+    weight: 2
+    region: content
+  field_card_media:
+    type: entity_reference_entity_view
+    label: above
+    settings:
+      view_mode: default
```

Comment:
Same here, we probably want a card view mode for media and to hide the title

Reply:
Great point, added a view mode in 2cee826749894cd5e4c8e5950a7dfe38b8e86f05.